docs/Install/Starting_Swarm_Learning_node.md

```diff
@@ -17,6 +17,8 @@ NOTE:
 |`--host-ip <IP address or DNS name>`|The IP address or DNS name of the host system on which this Swarm Network node is created.|None|
 |`--sn-ip <IP address or DNS name>`|The IP address or DNS name of the host system on which the Swarm Network node with which this Swarm Learning node must associate, while running.|None|
 |`--sn-api-port <port number>`|The host port for this Swarm Network node's API Server.|None|
+|`--sn-api-service <fqdn>:<port number>`|Fully Qualified Domain Name for the SN API Service of associated SN node. Here, Port number is optional.| |
 |`--sn-docker-name <container name>`|Docker container name for the associated Swarm Network node.|None|
 |`--sl-fs-port`|The host port for this Swarm Learning node's File Server.|None|
+|`--sl-fs-service <fqdn>:<port number>`|Fully Qualified Domain Name for the SL FS Service of associated SL node. Here, Port number is optional.| |
```

Comment:
-> Fully Qualified Domain Name and optional port for this Swarm Learning node's file service. Default: None